hrqb/tasks/employee_appointments.py

```diff
@@ -57,6 +57,9 @@ def get_dataframe(self) -> pd.DataFrame:
         libhr_df = self.named_inputs["ExtractQBLibHREmployeeAppointments"].read()
         depts_df = self.named_inputs["ExtractQBDepartments"].read()
 
+        # filter libhr data to active appointments, with position IDs
+        libhr_df = libhr_df[(libhr_df["Active"]) & ~(libhr_df["Position ID"].isna())]
```

Comment:
For those not super familiar with `pandas` filtering, this syntax breaks down to:

```python
libhr_df[   # filter this dataframe by the following conditions
    (libhr_df["Active"])  # filter where 'Active' column is True
    & ~(libhr_df["Position ID"].isna())  # filter where `Position ID` is not (the `~` operator) NULL
]
```